src/Tizen.NUI/src/public/BaseComponents/VectorGraphcis/Drawable.cs

```diff
@@ -0,0 +1,115 @@
+/*
+* Copyright(c) 2021 Samsung Electronics Co., Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+using System.ComponentModel;
+
+namespace Tizen.NUI.BaseComponents.VectorGraphics
+{
+    /// <summary>
+    /// Drawable is a object class for drawing a vector primitive.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Drawable : BaseHandle
+    {
+        /// <summary>
+        /// Creates an initialized drawable.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private Drawable() {}
+
+        internal Drawable(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Get the transparency value
+        /// </summary>
+        /// <returns>The transparency level</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float GetOpacity()
```

Comment:
Sorry for the frequent reviews.
In C# language, Get/Set methods are commonly expressed as `Property`.
In this case, `Opacity` can be a property, such as

```C#
public float Opacity
{
  get
  {
    // Add implement getter
  }
  set
  {
    // Implement setter
  }
}
```

Reply:
Changed